shell/platform/tizen/tizen_embedder_engine.h

```diff
@@ -20,8 +20,8 @@
 #include "flutter/shell/platform/tizen/channels/text_input_channel.h"
```

Comment:
Please change the file mode back to 644.

Reply:
ok